### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
I updated the documentation link and then thought that maybe it was easier to direct people to there instead of maintaining two different guides, but I am also open to reproducing the new getting started guide here instead

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
This also means we should update the link in the repository, and possibly delete the _docs_ folder?

**Review by jeromefroe — APPROVED** [review]
LGTM, @schallert is much more familiar with the project than I am though so would be good to get his thoughts as well

File: README.md [hunk 1]
```diff
@@ -14,12 +14,6 @@ More information:
...
 
-## Getting Started
-
-The following instructions serve as a quickstart to get an M3DB cluster up and running in your Kubernetes cluster. This
-setup is not for production use, as there's no persistent storage. More information on production-grade clusters can be
-found in our [docs][docs].
-
 ### Kubernetes Cluster Prerequisites
 
 The M3DB operator targets Kubernetes **1.11** and **1.12**. We generally aim to target the latest two minor versions
```
Comment (schallert) [inline README.md:19]: Can you also remove this entire paragraph?

**Review by schallert — APPROVED** [review]

**Comment by ChrisChinchilla** [comment]
@schallert Removed that paragraph, also removed the docs folder, this might break some build tasks in the main repo, but those tasks should be updated anyway if that's the case ;)

**Pull request closed without merging**